website/sidebars.js

```diff
@@ -30,7 +30,8 @@ module.exports = {
         'failing-closed',
         'mutation',
         'constrainttemplates',
-        'externaldata'
+        'externaldata',
+        'gator'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PhoenixRedflash/gatekeeper/pullRequest?prid=8495202)
```suggestion
        "gator"
```